features/mogelijkOnjuist.feature

```diff
@@ -0,0 +1,149 @@
+# language: nl
+Functionaliteit: Mogelijk onjuist
+  In de resource wordt aangegeven of bepaalde teruggegeven waarden in de resource mogelijk onjuist zijn. Waarden in de resource zijn mogelijk onjuist wanneer er onderzoek wordt gedaan naar de juistheid van onderliggende gegevens in de registratie.
+
+  Scenario: wanneer de status van een object in onderzoek is, zijn alle daaruit afgeleide properties van de resource mogelijk onjuist
+    Gegeven een resource adressen wordt opgevraagd
```

Comment:
Ik weet niet of het op de volgende manier duidelijker is:
```
Gegeven in een object nummeraanduiding is Status nummeraanduiding in onderzoek
Als de bijbehorende Adres resource wordt opgevraagd
Dan bevat het antwoord
...
```
Ik heb moeite met 'Gegeven een resource adressen wordt opgevraagd', omdat ik het gevoeld heb dat hier de path '/adressen' wordt bedoeld

Reply:
@MelvLee ik heb het nu gemaakt "Als de resource adressen wordt opgevraagd". Verwijst in zekere zin inderdaad naar endpoint /adressen, maar ik bedoel een specifieke resource. Dus /adressen/{adresIdentificatie}. Alleen heb ik nergens gedefinieerd welk adres dat specifiek is (met welke adresIdentificatie).
hoe zou jij dit in de feature beschrijven?

Reply:
In de titel van de scenario's heb je het over afgeleide properties. Ik zou dan de Als stap als volgt maken:
```
Als de afgeleide Adres resource wordt opgevraagd
```

Reply:
dank voor de tip. Heb het zo nu doorgevoerd in de feature